sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/api/Azure.ResourceManager.ContainerRegistry.net8.0.cs

```diff
@@ -3780,4 +3199,14 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ContainerRegistry.Models.SourceUploadDefinition>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ContainerRegistry.Models.SourceUploadDefinition>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class TlsPropertiesCertificate : Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTlsCertificateProperties, System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ContainerRegistry.Models.TlsPropertiesCertificate>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ContainerRegistry.Models.TlsPropertiesCertificate>
```

Comment:
same situation here, we rename this from `ContainerRegistryTlsCertificateProperties` to this.
I think `ContainerRegistryTlsCertificateProperties` is better.